FlightReservation/components/formComponent.js

```diff
@@ -0,0 +1,180 @@
+import React, { useState } from 'react';
+import {
+    StyleSheet,
+    View,
+    Text,
+    Image,
+    TextInput,
+    Pressable,
+} from 'react-native';
+
+function Form(props) {
+    const {validEmail,setValidEmail, validPassword, setValidPassword, changeForm, objValues, isLoginFormActive } = props;
+    const [isSeePassword, setSeePassword] = useState(true);
+
+    //const [validEmail, setValidEmail] = useState(true);
+    //const [validPassword, setValidPassword] = useState(true);
+
+    const changeSeePassword = () => {
+        setSeePassword(!isSeePassword);
+    };
+    const inputBorderColor = {
+        borderColorName: '#818181',
+        borderColorEmail: '#818181',
+        borderColorPassword: '#818181',
+    };
```

Comment:
can you refactor this?

Reply:
can you please make a more specific suggestion?

Reply:
We add the colors to file const